src/Test.UnitTests.Sarif.Converters/FortifyFprConverterTests.cs

```diff
@@ -40,5 +42,59 @@ public void FortifyFprConverter_Convert_TwoResultsWithNodeRefs()
         {
             RunTest("TwoResultsWithNodeRefs.fpr");
         }
+
+        [Fact]
+        public void FortifyFprConverter_Convert_ScanWithSeverityData()
+        {
+            RunTest("SdkScan.fpr");
+        }
+
+        [Fact]
+        public void FortifyFprConverter_GetFailureLevelFromRuleMetadata_MissingImpactProperty_ReturnsNone()
```

Comment:
>MissingImpactProperty_ReturnsNone [](start = 72, length = 33)

Thanks for such a well-named test!

I don't agree with what the test says. If `Impact` is missing from a Fortify rule definition, that feels to me like a SARIF rule definition that doesn't specify `defaultConfiguration.level`. So I think it should get the same treatment: it should correspond to a SARIF `result.level` of `"warning"` -- in which case you can omit it from the SARIF `result` object.